### DISCUSSION TITLE
Anyone willing to give a testimonial?

### DISCUSSION THREAD

**Discussion by blakeblackshear** [blakeblackshear/frigate#1866] [General]
I am working on a new website for Frigate, and I am hoping to have a few testimonial statements from users. If you are willing, please post a short statement here, and I will choose some of the responses to include. Thanks!

**Comment by PrplHaz4** [comment]
I use Frigate because, compared to other self-hosted NVRs, it is more accurate out of the box with higher quality events, easier on system resources and quick to setup while also being fully featured and highly configurable.

**Comment by pdecat** [comment]
_Frigate is awesome, and it's getting better and better!_

**Comment by Rodrigues-Alexandre** [comment]
Frigate gives you peace of mind without the burden of checking your cameras or spending money on expensive hardware. The integration with home assistant and the comprehensive user interface makes it easy to adopt by the whole family.

**Comment by eblohm** [comment]
Frigate has allowed me to remove all cloud subscriptions from my security cameras, without losing any sort of object detection features or recording history. Support is second to none. Highly recommended.

**Comment by shred86** [comment]
Frigate's high level of customizability, fast object detection and tight integration with Home Assistant creates the perfect open source, locally controlled, security camera system.

**Comment by haggercody** [comment]
Frigate has helped me reduce hours of false detections from my hard drive and saved me maybe as much time scouring through said, uneventful, footage.  Ok maybe not that much, but seriously, zero false detections.

**Comment by madmic1314** [comment]
Frigate is amazing, written by a super genius (as I have no idea on this stuff!) who is very supportive of the community. And best of all? It's open source, free and local so no cloud!!!

**Comment by mcondren** [comment]
I have four security cameras covering my property and using the built-in motion detection provided by the camera manufacturer resulted in multiple false positives every day. To the point where I wasn’t even checking the alerts anymore, which completely defeats the purpose of having security cameras. I switched to Frigate, and weeks go by without a single false detection.

**Comment by blakeblackshear** [comment]
Thanks to everyone who submitted. The new website is live at https://frigate.video

I may rotate the testimonials periodically.

**Reply by haggercody** [reply]
woohoo i'm famous! 

nice presentation, also :)
